WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -7,6 +7,7 @@
 static NSInteger HideAllSitesThreshold = 6;
 static NSTimeInterval HideAllSitesInterval = 2.0;
 static NSInteger HideSearchMinSites = 3;
+static NSTimeInterval AutoSyncInterval = 300;
```

Comment:
Why are we limiting this to 5 minutes? I'm not sure if we should do it or if that's the right interval, but it seems unrelated to the sync locking changes and I'd prefer to leave it out of this PR.

Since we just added #12377 to de-duplicate blogs on sync, I'd rather sync too often, at least until this issue is sorted out.